core/src/actions/destinations.ts

```diff
@@ -325,101 +325,27 @@ export class DestinationView extends AuthenticatedAction {
 
 export class DestinationExport extends AuthenticatedAction {
   name = "destination:export";
-  description = "export the members of the groups tracked by this destination";
+  description =
+    "export the member or members of the groups tracked by this destination";
   permission: ActionPermission = { topic: "destination", mode: "write" };
   inputs = {
-    id: { required: true },
-  } as const;
-
-  async runWithinTransaction({
-    params,
-  }: {
-    params: ParamsFrom<DestinationExport>;
-  }) {
-    const destination = await Destination.findById(params.id);
-    await destination.exportMembers();
-    return { success: true };
-  }
-}
-
-export class DestinationRecordPreview extends AuthenticatedAction {
-  name = "destination:recordPreview";
```

Comment:
I think I don't understand how there can still be a Record preview card on the Destination/data page, using the nouns the Destination uses, without this action

Reply:
Making a `RecordView` call gives us all the data this did, but more consistently.